### PR TITLE
(PDB-1767) Update release notes for version 3.0.1

### PR DESCRIPTION
**Pull request by mullr**

### CODE REVIEW

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -10,6 +10,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+* As part of the PuppetDB 3.0.0 release, the puppetdb-terminus package was
+  renamed to puppetdb-termini for consistency with other Puppet Labs products. A
+  flawed dependency in the puppetdb-termini package prevented installation of
+  the older 2.3.x puppetdb-terminus package on RPM-based platforms. This release
+  fixes that dependency, allowing the older version to be installed.
+  ([PDB-1760](https://tickets.puppetlabs.com/browse/PDB-1760))
+
+  To fully correct the problem, the existing 3.0.0 puppetdb and
+  puppetdb-terminus packages will be removed from the Puppet Collection 1 yum
+  repository.
```
Comment (wkalt) [inline documentation/release_notes.markdown]: and replaced with new ones

File: documentation/release_notes.markdown [hunk 2]
```diff
@@ -10,6 +10,30 @@ canonical: "/puppetdb/latest/release_notes.html"
 [upgrading]: ./api/query/v4/preparing-for-3.0.html
 [puppetdb-module]: https://forge.puppetlabs.com/puppetlabs/puppetdb
 
+3.0.1
+-----
+
+PuppetDB 3.0.1 is a bugfix release to fix packaging problems in PuppetDB 3.0.0.
```
Comment (senior) [inline documentation/release_notes.markdown]: Maybe we should specifically put YUM or versions of RHEL/CentOS affected?

File: documentation/release_notes.markdown [hunk 3]
```diff
@@ -10,6 +10,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  the older 2.3.x puppetdb-terminus package on RPM-based platforms. This release
+  fixes that dependency, allowing the older version to be installed.
+  ([PDB-1760](https://tickets.puppetlabs.com/browse/PDB-1760))
+
+  To fully correct the problem, the existing 3.0.0 puppetdb and
+  puppetdb-terminus packages will be removed from the Puppet Collection 1 yum
+  repository.
+
+* When upgrading from a 3.0.x package to a 3.0.y package, don't run the 2.3->3.0
+  migration script. ([PDB-1763](https://tickets.puppetlabs.com/browse/PDB-1763))
```
Comment (ajroetker) [inline documentation/release_notes.markdown]: The 2.x->3.x migration script now only runs on upgrades from PuppetDB <3.0.1

File: documentation/release_notes.markdown (hunk 2 above)
Comment (ajroetker) [inline documentation/release_notes.markdown]: Agreed

File: documentation/release_notes.markdown (hunk 3 above)
Comment (mullr) [inline documentation/release_notes.markdown]: < 3.0.0 you mean?

Comment (ajroetker) [inline documentation/release_notes.markdown]: Yeah, that one

**Comment by mullr** [comment]
@senior @wkalt @ajroetker updated

**Comment by senior** [comment]
:+1:

**Comment by ajroetker** [comment]
:+1:

**Comment by mullr** [comment]
^^ transients

**Pull request merged**